swagger-api-client/publish.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -e
+
+# sbt publish publishes libs to Artifactory for the scala version sbt is running as.
+# sbt +publish publishes libs to Artifactory for all scala versions listed in crossScalaVersions.
+SNAPSHOT=$(( "$TRAVIS_PULL_REQUEST" == "false" && "$TRAVIS_BRANCH" == "develop" ? "SNAP" : "" ))
+
+echo "getting swagger codegen jar..."
+wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-2.2.3.jar -O swagger-codegen-cli.jar
+echo "generating Sam Java Client API..."
+java -jar swagger-codegen-cli.jar generate -l java --input-spec ./src/main/resources/swagger/api-docs.yaml --output generated
+
+COMMIT_HASH="${TRAVIS_COMMIT:0:7}"
+VERSION_HASH="1.0-${COMMIT_HASH}${SNAPSHOT}"
```

Comment:
So I want to grab the version number from `project/Version.scala` which sets `val baseModelVersion = "0.1"`
I was told that sbt had a way to do this?

Reply:
You might want inspect:
http://www.scala-sbt.org/1.x/docs/Howto-Inspect-the-Build.html